schema/ridl/tokenizer.go

```diff
@@ -1,15 +1,33 @@
 package ridl
 
-func tokenize(src []byte) ([]token, error) {
+func tokenize(src []byte) ([]token, map[int][]string, error) {
```

Comment:
can we return `map[int]string`, ie. full comment string for each line number?

Reply:
Makes sense. Done :)